config/pipeline.yaml

```diff
@@ -1825,6 +1825,135 @@ scheduler:
     platforms:
       - qemu-x86
 
+  - job: rt-tests-cyclicdeadline
+    event: &kbuild-gcc-12-arm64-preempt_rt-node-event
+      channel: node
+      name: kbuild-gcc-12-arm64-preempt_rt
+      result: pass
+    runtime: *runtime-lava-collabora
+    platforms: &kbuild-gcc-12-arm64-preempt_rt-platforms
+      - mt8183-kukui-jacuzzi-juniper-sku16
+      - mt8186-corsola-steelix-sku131072
+      - sc7180-trogdor-kingoftown
```

Comment:
Do those really work when not using a chromebook kbuild?

Reply:
I don't think so. I've changed the platforms without chromebook fragment.